### DISCUSSION TITLE
File picker doesn't show full path

### DISCUSSION THREAD

**Discussion by DoctorRainer** [helix-editor/helix#11644] [Ideas]
![CleanShot 2024-09-03 at 20 02 26@2x](https://github.com/user-attachments/assets/9956ab97-4717-4fe0-a110-6449531375f8)

This is how it looks like.

I work on a big project with long names and huge nesting. So, when I'm searching for files with similar names, I can't see what applications or packages are they in, especially important is the starting part. Maybe we could change the way it folds to expose at least the starting part? Or maybe it's better to display full path to a file in a separate place?

**Comment by leo-unglaub** [comment]
You can use `<Ctrl>+t` to toggle the preview, that way you get more space for the filename. Maybe that helps a little bit.

**Comment by DoctorRainer** [comment]
@leo-unglaub oh, it makes it much better, we almost have no files that overflow but we still do, which is not critical. A bit inconvenient but at least it makes my job easier
